src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public class Task {
```

Comment:
Suggestion:
make Task class into an abstract class as it is not instantiated anywhere.